### PR TITLE
SRCH-387 make BingV7 default search engine

### PR DESCRIPTION
**Pull request by MothOnMars**
This includes a migration to set `BingV7` as the default value for `affiliates.search_engine`. It also updates the VCR cassettes to use BingV7.

### CODE REVIEW

**Comment by MothOnMars** [comment]
@peggles2 , CircleCI has been a little wonky this morning, and I noticed that it hasn't actually tested this PR yet. I'm going to close & re-open this PR in hopes that it will trigger a test run.

File: db/migrate/20190205200912_change_default_search_engine_to_bing_v7.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class ChangeDefaultSearchEngineToBingV7 < ActiveRecord::Migration
+  def up
+    change_column_default :affiliates, :search_engine, 'BingV7'
+  end
+
+  def down
+    change_column_default :affiliates, :search_engine, 'BingV6'
+  end
+end
```
[the remark this answers is not in the document]
Comment (peggles2) [reply db/migrate/20190205200912_change_default_search_engine_to_bing_v7.rb]: How will this impact the application migrating and changing code at the same time?  Your tests may fail expecting BingV7 before it gets migrated.  You may want to make this a separate push. Run the Migration, then push the code.

[the remark this answers is not in the document]
Comment (MothOnMars) [reply db/migrate/20190205200912_change_default_search_engine_to_bing_v7.rb]: There are no code changes in this commit - only the migration and spec changes.

[the remark this answers is not in the document]
Comment (MothOnMars) [reply db/migrate/20190205200912_change_default_search_engine_to_bing_v7.rb]: @peggles2 , parentheses around arguments should not be used in migrations as they are using the Rails DSL:
https://github.com/rubocop-hq/ruby-style-guide#syntax

File: spec/models/affiliate_spec.rb [hunk 2]
```diff
@@ -16,21 +16,35 @@
...
-
-    it { is_expected.to have_db_index(:active_template_id) }
-    it { is_expected.to have_db_index(:template_id) }
+    describe 'columns' do
+      it do
+        is_expected.to have_db_column(:i14y_date_stamp_enabled).
+          of_type(:boolean).with_options(default: false, null: false)
+      end
+      #The active_template_id column has been deprectated. It will be dropped in a future migration.
+      it { is_expected.to have_db_column(:active_template_id).of_type(:integer) }
```
[the remark this answers is not in the document]
Comment (MothOnMars) [reply spec/models/affiliate_spec.rb]: will do 👍

File: db/migrate/20190205200912_change_default_search_engine_to_bing_v7.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (peggles2) [reply db/migrate/20190205200912_change_default_search_engine_to_bing_v7.rb]: Yeah reason why is rubcop is complaining about it. What I would do is maybe look into updating the rubcop.yml to not include this. Also saw https://apidock.com/rails/ActiveRecord/ConnectionAdapters/SchemaStatements/change_column_default it had parentesis. However I personally never put parenthesis in migrations and well you know was not a fan of the parenthesis. I think looks cleaner with out.

[the remark this answers is not in the document]
Comment (MothOnMars) [reply db/migrate/20190205200912_change_default_search_engine_to_bing_v7.rb]: OK - I'll add `db/` as an exclusion path for that cop.

**Pull request merged**